api/preprints/serializers.py

```diff
@@ -113,7 +113,7 @@ class PreprintSerializer(TaxonomizableSerializerMixin, MetricsSerializerMixin, J
     date_published = VersionedDateTimeField(read_only=True)
     original_publication_date = VersionedDateTimeField(required=False, allow_null=True)
     custom_publication_citation = ser.CharField(required=False, allow_blank=True, allow_null=True)
-    doi = ser.CharField(source='article_doi', required=False, allow_null=True)
+    doi = ser.CharField(source='article_doi', required=False, allow_null=True, allow_blank=True)
```

Comment:
Ah, something I learned today, `blank` is validation which doesn't need migration if changed, while `null` is DB related which needs migration if changed.